extra/Addons/Macros/SelectionStartEnd.lua

```diff
@@ -0,0 +1,114 @@
+﻿-- Selection Start/End with hot-keys Ctrl+( / Ctrl+) or using Menu
+local Color = far.AdvControl(far.Flags.ACTL_GETCOLOR,far.Colors.COL_HELPTOPIC)
```

Comment:
If we removed `Hint`, `Color` is not needed either unless I'm missing something.